test/integration/cds_integration_test.cc

```diff
@@ -160,7 +164,7 @@ TEST_P(CdsIntegrationTest, CdsClusterUpDownUp) {
   ASSERT_TRUE(codec_client_->waitForDisconnect());
 
   // Tell Envoy that cluster_1 is back.
-  EXPECT_TRUE(compareDiscoveryRequest(Config::TypeUrl::get().Cluster, "42", {}, {}, {}));
+  EXPECT_TRUE(compareDiscoveryRequest(Config::TypeUrl::get().Cluster, "42", {}, {}, {}, false));
```

Comment:
In many of the `compareDiscoveryRequest` calls the PR adds the 6th arg as false (which IIUC should be false by default). Would it be better to remove it from these calls? (less changes across all integration tests)